lib/Field.js

```diff
@@ -640,8 +641,86 @@ class Field {
 
     return this.default;
   }
+
+  _objectsToExpressions(option, args) {
+    const expressions = [];
+
+    args.forEach(arg => {
+      if (
+        option !== 'and' && // `and` already supports objects
+        option !== 'not' && // `not` already supports objects
+        option !== 'or' && // `or` already supports objects
+        isObject(arg) &&
+        !Array.isArray(arg)
+      ) {
+        Object.entries(arg).forEach(([field, value]) => {
+          if (!field.startsWith('_$_')) {
+            expressions.push({ [`_$_${option}`]: [field, value] });
+          }
+        });
+      }
+    });
+
+    return expressions;
+  }
+
+  addOption(option, args) {
+    args = args.map(arg => {
+      return { [this.name]: arg };
+    });
+    const expressions = this._objectsToExpressions(option, args);
+
+    return expressions.length
+      ? { _$_and: expressions }
+      : { [`_$_${option}`]: args };
+  }
 }
 
+const whereOptions = {
+  and: 'and',
```

Comment:
We don't need `and`, `or` and `not` here, since `{ where: Fields.id.and(...) }` doesn't really work. I think for these we'll still need to keep it in `Where`:
```js
Model.fetch({ where: Model.where.or([Model.fields.id.equal(1), Model.fields.id.equal(2)]) });
```
And that means that [these checks](https://github.com/knorm/knorm/pull/251/files#diff-d2f2478a2a2643b95d68b7dd5a4af056R650) will not be needed anymore

Reply:
@joelmukuthu I totally forgot to remove it, I initially had only some of them like equal, notEqual etc and then recopied the entire thing from where again and committed it like as it is. I will remove it. No issues.